src/styles/main.scss

```diff
@@ -1,3 +1,60 @@
+$--day-block-width: 100px;
+$--day-block-height: 100px;
+$days: ("mon": 0, "tue": 1, "wed": 2, "thu": 3, "fri": 4, "sat": 5, "sun": 6);
+
 body {
   margin: 0;
 }
```

Comment:
move this to other file, it doesn't relate to the calendar

Reply:
✅Done